peachjam/views/causelist.py

```diff
@@ -40,8 +41,12 @@ class CauseListListView(TemplateView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-
-        context["court_classes"] = CourtClass.objects.prefetch_related("courts")
+        # get only court classes and courts that have cause lists, and cache the result
+        court_classes_with_cause_lists = cache.get_or_set(
```

Comment:
remember, we use the file system as the cache. so the cached values have to be serialisable. i don't think it will work to try to cache a queryset.

Reply:
ok, so we need to fully build out the object along with the corresponding courts?

Reply:
what about casting the queryset to a list, is that enough or should I also break down the individual courtclasses  and related courts

Reply:
It looks like you can cache a queryset, no idea how that works: https://docs.djangoproject.com/en/4.2/ref/models/querysets/#pickling-querysets